src/scss/grommet-core/_objects.menu.scss

```diff
@@ -3,6 +3,19 @@
 .menu {
   position: relative;
   @include inuit-font-size($menu-font-size, inherit);
+  
+  .header & {
+    
+    &--controlled,
+    &__control {
+      height: 100%;
+    }
+    
+    &__control {
+      line-height: double($inuit-base-spacing-unit);
```

Comment:
I've updated the component to set the line-height of the label elements to the height of their container - I couldn't find a pure css way to do this, since the drop is inserted outside the header when the menu is clicked